### ISSUE TITLE
Add "IPDS_id" field to the gran tsv?

### ISSUE THREAD

**Issue by jread-usgs** [USGS-R/grantools#11]
Could be either a record ID (e.g., 061003 for `laketemps`) or "none" (or NA?)

Will help support future checks for `.onAttach` disclaimer. Also, will help address concerns from @rmhirsch49

**Comment by lawinslow** [comment]
We could also think about adding functionality to grantools. It wouldn't be hard to have it inject the disclaimer on any package not part of a "IPDS approved" list.

**Comment by lawinslow** [comment]
Wow, I had been watching your laketemps stuff come through, I didn't realize this was on grantools. So let me rephrase that, good idea. Could either check or just inject disclaimer as needed.

**Comment by jread-usgs** [comment]
would be nice to inject if needed. that makes it so we wouldn't have to show everyone how to do it.

**Comment by lawinslow** [comment]
yeah, I know exactly where it would go in the workflow. The downloads from Github are uncompressed before being built. That's the moment to add the file. Would get us most of the way there.
